src/common/containers/GameArea.js

```diff
@@ -522,14 +542,14 @@ export class GameArea extends Component {
                 onPrevStep={this.handlePrevTutorialStep}
               >
                 <EndTurnButton
-                  player={this.props.player}
+                  player={this.props.isSandbox ? this.props.currentTurn : this.props.player}
```

Comment:
`this.props.isSandbox ? this.props.currentTurn : this.props.player`

is repeated multiple times. Maybe pull this into a variable named `player` so the logic is in one place.

Reply:
I'll fix this after I merge in the GameArea refactor PR – I don't want to make the diff too messy right now.